.github/workflows/deploy-testrunner-integration-streamer.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow is used to deploy GCP cloud functions from source
+# For more information see: https://github.com/google-github-actions/deploy-cloud-functions
+
+name: Deploy TestRunner Results Streamer Function
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy-testrunner-streamer-from-source:
+
+    runs-on: self-hosted
+    env:
+      DEPLOY_ENV: INTEGRATION
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Configure secrets, SA, Project ID
```

Comment:
opt: a little wordy for me. How about `Configure secrets and project`?

Reply:
Simplified comments as suggested.